Library/Homebrew/blacklist.rb

```diff
@@ -81,5 +81,9 @@ def blacklisted?(name)
     A binary installer is available:
       https://www.haskell.org/platform/mac.html
     EOS
+  when "arm" then <<-EOS.undent
+    Arm has been renamed to Nyx (https://nyx.torproject.org/) and is available through PyPI.
```

Comment:
Nitpick: looks like `arm` prefers the undercover "a" in its name.

Reply:
Good catch - updated.